codes/classical/bits/random_code.yml

```diff
@@ -0,0 +1,128 @@
+
+  # # This field that starts a list of strings, indexed by a dash; do not leave it
+  # # empty. If empty, comment out.
+  # #encoders:
+  #   - 'Specific description of a process that makes the states, usually for quantum codes.'
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # # Not all fields are indexed by a dash
+  # #transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # #general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 'Ball-Collision Decoding \cite{doi:10.1007/978-3-642-22792-9_42}'
+             Finiasz and Sendrier (FS-ISD Decoding)\cite{doi:10.1007/978-3-642-10366-7_6'
+  #   - 'Details about how syndrome measurements are done; discuss overhead, if applicable.'
+  #   - 'MWPM decoding algorithm \cite{doi:good-paper} with ... overhead.'
+  #   - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # #fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  # #code_capacity_threshold:
+  #   - '\(1.5%\) error-correction threshold against some noise with *noiseless* decoder of some complexity \cite{arxiv:paper}.'
+  #   - '\(5.2\%\) asymptotic upper bound from some estimation algortihm (Monte Carlo, e.g.) that may or may not have a feasible decoder.'
+
+  # #threshold:
+  #   - '\(0.3\%\) error-correction threshold ... with *noisy* ... decoder of some complexity \cite{doi:good-paper}.'
+  #   - '\(10^{-5}\) computational threshold using concatenated scheme under ... noise with overhead of ... '
+  #   - '\(4.5\%\) asymptotic lower bound on fault-tolerant threshold with ... noise, ... overhead, and ... complexity.'
+  #   - '\(6.7\%\) threshold using magic-state distillation protocols assuming ... Clifford gates.'
+
+  # # Propose a field that you feel is applicable to a large and interesting class of codes
+  # #some_other_field:
+  # # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Distributed Storage Systems \cite{doi: 10.1109/TPDS.2008.251}'
+
+notes:
+    - 'Code features refer to the asymptotic limit, i.e. The code approach the minimum distance with probability 1 as N goes to infiity.'
```

Comment:
"The code approach the minimum distance with probability 1 as N goes to infiity." 
-> 
"We consider ensembles of codes of increasing length $n$ and cardinality $M=nR$, and argue about the limiting parameters of the typical or average codes in the ensemble as $n\to\infty$."